### ISSUE TITLE
config file

### ISSUE THREAD

**Issue by e-sr** [e-sr/moonboard#16]
It would be nice to have a config file containing the information about the led stripes type and the led layout.

- the config file should be a json or yml file
- the led layout sould be a string naming a provided layout or a 18x11 array providing the led addresses

see also #3

**Comment by 8cH9azbsFifZ** [comment]
Have hacked an implementation with Strip driver and JSON. My board is complete by now :)

We can discuss on a successive merge strategy of my fork, once my stuff is stabilized.

**Comment by marcelbesoeu** [comment]
8cH9azbsFifZ, will your hacked version work on  raspberry pi 3B+ or I have to move to Rapi W Zero?

**Comment by grasnag** [comment]
I use 8cH9azbsFifZ version on 3B+

ti 8.12.2020 klo 21.14 marcelbesoeu <notifications@github.com> kirjoitti:

**Comment by marcelbesoeu** [comment]
I use 3B+, fresh install "Raspberry Pi OS with desktop and recommended software" and tried e-sr, 8cH9azbsFifZ with requirements loaded and without - loaded only minimal packages neded to run "moonboard_BLE_service.py" without errors. No success. 
My phone is working with original MonnBoard led system without problems. I have my own Monnboard with all 2017 holds and LED sytem but want "LEDize" my training wall as well.

**Comment by 8cH9azbsFifZ** [comment]
> 8cH9azbsFifZ, will your hacked version work on raspberry pi 3B+ or I have to move to Rapi W Zero?

Yes - it works.

**Comment by 8cH9azbsFifZ** [comment]
> I use 3B+, fresh install "Raspberry Pi OS with desktop and recommended software" and tried e-sr, 8cH9azbsFifZ with requirements loaded and without - loaded only minimal packages neded to run "moonboard_BLE_service.py" without errors. No success.

Whoops there was a typo. Did you try:
https://github.com/8cH9azbsFifZ/moonboard/blob/master/doc/OVERVIEW-SOFTWARE.md
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/8cH9azbsFifZ/moonboard/master/install/install.sh)"

Just tested the installation once again here - it works with latest raspbian.

**Comment by marcelbesoeu** [comment]
Yes - I tried install.sh.  The same: app Is coneccting one time, but holds list is not sent and disconnect.
I will make clean install and try connect from diferent phone.

**Comment by grasnag** [comment]
Are you using IOS or android?

**Comment by marcelbesoeu** [comment]
Android on S10e.

**Comment by marcelbesoeu** [comment]
I found that original Moonbord LED BOX Bluetooth name  is not "Moonboard A" but "MoonBoard A". 
Still no success with connection from app.
Led driving using WS2811 led's works.
